$OpenBSD: patch-iwyu_location_util_h,v 1.1 2017/02/10 09:39:21 ajacoutot Exp $

Update to reflect changes in Clang.

In r283766 started using unique_ptr instead of raw pointers in the
late-parsed templates map.

In r283815 changed FileEntry::Name from raw pointer to StringRef.

--- iwyu_location_util.h.orig	Sat Jan 14 19:45:02 2017
+++ iwyu_location_util.h	Sat Jan 14 19:45:32 2017
@@ -78,8 +78,7 @@ inline bool IsBuiltinFile(const clang::FileEntry* file
 // IsBuiltinOrCommandLineFile(file) returns true if it's either of the
 // two cases.
 inline bool IsBuiltinOrCommandLineFile(const clang::FileEntry* file) {
-  return IsBuiltinFile(file) ||
-         (strcmp(file->getName(), "<command line>") == 0);
+  return IsBuiltinFile(file) || file->getName().equals("<command line>");
 }
 
 // When macro args are concatenated e.g. '#define CAT(A, B) A##B', their
